src/goto-instrument/dump_c_class.h

```diff
@@ -135,13 +137,15 @@ class dump_ct
     const bool use_all_headers,
     const bool include_harness,
     const namespacet &_ns,
+    const optionalt<irep_idt> _stub_name,
```

Comment:
I'm not sure why this is being added as an argument to `dump_c`

Reply:
Thanks! As I am extending the dump-c infrastructure, I thought it would be better to keep a unified interface. Do you have any ideas about how to improve that?